dotnet/packages/Microsoft.TeamsAI/Microsoft.TeamsAI/Application/StreamingChannelData.cs

```diff
@@ -41,5 +41,11 @@ public class StreamingChannelData
         /// </summary>
         [JsonProperty(PropertyName = "feedbackLoopEnabled")]
         public bool? feedbackLoopEnabled { get; set; }
+
+        /// <summary>
+        /// Represents the type of feedback loop. Set to "default" by default. It can be set to one of "default" or "custom".
+        /// </summary>
+        [JsonProperty(PropertyName = "feedbackLoopType")]
+        public string? feedbackLoopType { get; set; }
```

Comment:
do we need to set default here?

Reply:
Nope, since it will be set to null by default.